### PR TITLE
Microbe updates on postrelease/50

### PR DESCRIPTION
**Pull request by nds**

### CODE REVIEW

File: protists/archives.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
+The following archive sites are available to access previous versions of data:
 
-Archive of release 40 of EnsemblProtists: [eg40-protists.ensembl.org](http://eg40-protists.ensembl.org) (July 2018)
-
-Archive of release 37 of EnsemblProtists: [eg37-protists.ensembl.org](http://eg37-protists.ensembl.org) (October 2017)
+ - Release 49, December 2020  [eg49-protists.ensembl.org](http://eg49-protists.ensembl.org)
+ - Release 45, September 2019 [eg45-protists.ensembl.org](http://eg45-protists.ensembl.org)
+ - Release 40, July 2018      [eg40-protists.ensembl.org](http://eg40-protists.ensembl.org)
+ - Release 37, October 2017   [eg37-protists.ensembl.org](http://eg37-protists.ensembl.org)
+~
```
Comment (azangru) [inline protists/archives.md:9]: One remaining tilde to delete 😄

Comment (nds) [reply protists/archives.md]: Done :)

File: bacteria/access.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
 - *Search for a gene* - type the name of a gene or other identifier into the search box above
 - *Find a genome* - click in the 'browse a genome' box above and start typing your genome name to find matching genomes
 - [View full list of all Ensembl Bacteria species](/species.html)
-- [Access Ensembl Bacteria programmatically](/info/data/api.html)
+- [Access Ensembl Bacteria programmatically] (/info/data/api.html)
```
Comment (azangru) [inline bacteria/access.md:4]: Sorry, one more thing. This space will break the link. Markdown syntax requires no space between the text of the link in square brackets and the url of the link in parentheses.

Comment (nds) [reply bacteria/access.md]: Oops sorry. Done.

**Review by azangru — APPROVED** [review]

**Pull request merged**